window/Cargo.toml

```diff
@@ -82,7 +82,7 @@ zbus = "4.2"
 zvariant = "4.0"
 
 smithay-client-toolkit = {version = "0.19", default-features=false, optional=true}
-wayland-backend = {version="0.3.5", features=["client_system", "rwh_06"]}
+wayland-backend = {version="0.3.5", features=["client_system", "rwh_06"], optional=true}
```

Comment:
I think you also need to add `wayland-backend` to the list of features up on line 18 for this to work correctly

Reply:
That would explain all of the build failure emails I got, updated!